resources/user-agents/browsers/misc/smarttv-webbrowser.json

```diff
@@ -29,6 +30,72 @@
           }
         }
       ]
+    },
+    {
+      "userAgent": "SmartTV WebBrowser Webkit",
```

Comment:
These patterns come from the `safari-x-.json` files.  I looked around a bit to see if what these TVs use is actually safari, which doesn't seem to be the case. It sounds like it's definitely a webkit based browser, but isn't necessarily "Safari" since it's an embedded OS.  I thought it was misleading to keep it as "Safari" so moved them over to this file to name them "SmartTV WebBrowser" which I think is more indicative of what they are.